orangecontrib/text/tests/test_preprocess.py

```diff
@@ -47,22 +49,39 @@ def test_preprocess_one_sentence_lowercase_stopwords(self):
         self.assertEqual(sorted(corpus),
                          sorted(['Human', 'interface', 'lab', 'machine']))
 
-    def test_preprocess_corpus_min_df(self):
-        p = Preprocessor(lowercase=True, stop_words=['for', 'a', 'of', 'the', 'and', 'to', 'in'], min_df=2)
+    def test_preprocess_corpus_int_df(self):
+        p = Preprocessor(lowercase=True, min_df=1, max_df=4)
         corpus = p(self.corpus)
-        correct = [['human', 'interface', 'computer'],
-                   ['survey', 'user', 'computer', 'system', 'response', 'time'],
-                   ['eps', 'user', 'interface', 'system'],
-                   ['system', 'human', 'system', 'eps'],
-                   ['user', 'response', 'time'],
-                   ['trees'],
-                   ['graph', 'trees'],
-                   ['graph', 'minors', 'trees'],
-                   ['graph', 'minors', 'survey']]
-        self.assertEqual(len(corpus), len(correct))
-        for i, j in zip(corpus, correct):
-            self.assertEqual(sorted(i), sorted(j))
-
+        correct = [
+            ['interface', 'computer'],
+            ['a', 'survey', 'user', 'computer', 'system', 'response', 'time'],
+            ['the', 'eps', 'user', 'interface', 'system'],
+            ['system', 'and', 'system', 'eps'],
+            ['relation', 'user', 'response', 'time'],
+            ['the', 'trees'],
+            ['the', 'trees'],
+            ['minors', 'iv', 'widths', 'trees', 'and'],
+            ['minors', 'a', 'survey']
+        ]
+        self.assertEqual(corpus, correct)
+
+    def test_preprocess_corpus_float_df(self):
+        p = Preprocessor(lowercase=True, min_df=0.1, max_df=0.8)
```

Comment:
`max_df` here does not filter out anything. Can you add a test, where `max_df` given as `float` also filter out something?